### PR TITLE
Created a project file for Visual studio 2017

### PR DESCRIPTION
**Pull request by tarung65**

### CODE REVIEW

File: PowerEditor/visual.net/notepadPlus.vs2017.vcxproj [hunk 1]
```diff
@@ -0,0 +1,682 @@
...
+    <ProjectName>Notepad++</ProjectName>
+    <ProjectGuid>{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}</ProjectGuid>
+    <RootNamespace>Notepad++</RootNamespace>
+    <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Unicode Release|Win32'" Label="Configuration">
+    <ConfigurationType>Application</ConfigurationType>
+    <PlatformToolset>v140_xp</PlatformToolset>
```
Comment (SinghRajenM) [inline PowerEditor/visual.net/notepadPlus.vs2017.vcxproj]: It does not use VS2017 compiler. Shouldn't be v141_xp?

File: PowerEditor/visual.net/notepadPlus.vs2017.vcxproj [hunk 2]
```diff
@@ -0,0 +1,682 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (SinghRajenM) [inline PowerEditor/visual.net/notepadPlus.vs2017.vcxproj]: Shouldn't be ToolsVersion="15.0"?

File: PowerEditor/visual.net/notepadPlus.vs2017.vcxproj [hunk 3]
```diff
@@ -0,0 +1,682 @@
...
+      <Configuration>Unicode Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <ProjectName>Notepad++</ProjectName>
+    <ProjectGuid>{FCF60E65-1B78-4D1D-AB59-4FC00AC8C248}</ProjectGuid>
+    <RootNamespace>Notepad++</RootNamespace>
+    <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>10.0.15063.0</WindowsTargetPlatformVersion>
```
Comment (tarung65) [inline PowerEditor/visual.net/notepadPlus.vs2017.vcxproj]: I just retarget the project and my project compiled it compile but didn't compile earlier

**Comment by tarung65** [comment]
Created

**Pull request closed without merging**